Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -1490,33 +1494,30 @@ extension SourceKitServer {
   /// Handle a workspace/symbol request, returning the SymbolInformation.
   /// - returns: An array with SymbolInformation for each matching symbol in the workspace.
   func workspaceSymbols(_ req: WorkspaceSymbolsRequest) async throws -> [WorkspaceSymbolItem]? {
-    let symbols = findWorkspaceSymbols(
-      matching: req.query
-    ).map({ symbolOccurrence -> WorkspaceSymbolItem in
-      let symbolPosition = Position(
-        line: symbolOccurrence.location.line - 1,  // 1-based -> 0-based
-        // FIXME: we need to convert the utf8/utf16 column, which may require reading the file!
-        utf16index: symbolOccurrence.location.utf8Column - 1
-      )
-
-      let symbolLocation = Location(
-        uri: DocumentURI(URL(fileURLWithPath: symbolOccurrence.location.path)),
-        range: Range(symbolPosition)
-      )
-
-      return .symbolInformation(
-        SymbolInformation(
-          name: symbolOccurrence.symbol.name,
-          kind: symbolOccurrence.symbol.kind.asLspSymbolKind(),
-          deprecated: nil,
-          location: symbolLocation,
-          containerName: symbolOccurrence.getContainerName()
-        )
-      )
-    })
+    let symbols = findWorkspaceSymbols(matching: req.query).map(WorkspaceSymbolItem.init)
     return symbols
   }
 
+  func workspaceTests(_ req: WorkspaceTestsRequest) async throws -> [WorkspaceSymbolItem]? {
+    let testSymbols = workspaces.flatMap { (workspace) -> [SymbolOccurrence] in
+      guard let index = workspace.index else {
+        return []
+      }
+      let xctestSubclasses = index.occurrences(ofUSR: "c:objc(cs)XCTest", roles: [.baseOf])
```

Comment:
The superclass to look for is named "XCTest**Case**". The class named plain "XCTest" is its superclass, but it's abstract and we only search for subclasses of the former.